examples.md

```diff
@@ -657,3 +658,21 @@ steps:
       restore-keys: |
         ${{ runner.os }}-mint-
 ```
+
+## * - Bazel
+
+```yaml
+- name: Cache Bazel
+  uses: actions/cache@v3
+  with:
+    path: |
+      ~/.cache/bazel # Linux
+      /private/var/tmp/_bazel_runner/ # macOS
```

Comment:
Create separate examples for linux and mac and remove todo for windows if you're unsure of that. You can take reference of how it is done for deno above.

Reply:
Great idea - done! Thank You.

Reply:
Why separate examples? Most people will have the same job running in a matrix of platforms, so it might be useful to have a "real world example" here.

Reply:
@PiotrSikora  we generally avoid showing os matrices in examples. some people would want to just use macos or ubuntu and hence this is better.

Reply:
Right, but even if they want to use it only on a single OS, then having non-existing directory here shouldn't cost anything.